### PR TITLE
Feature: READMEを整備する

### PR DESCRIPTION
**Pull request by krmr73**
## WHAT
- READMEを更新した
- English versionを追加した

### CODE REVIEW

**Comment by krmr73** [comment]
@ciaran-regan-ie I would like you to check the English version of the writing. And please let me know if there is anything additional I should write about.

File: README-en.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
+# UrnBasedDynamicNetworkModel
+[日本語](README.md) | English
+
+## Abstruct
```
Comment (ciaran-regan-ie) [inline README-en.md:4]: Should say
Abstract

File: README-en.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+This is the repository for the article "Simulating Emergence of Novelties Using Agent-Based Models", which was submitted to PLOSONE.
```
Comment (ciaran-regan-ie) [inline README-en.md:5]: I think PLOS ONE is two words

File: README-en.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+
+```sh
+$ julia --proj
+julia> <]>
+(AROB2023) pkg> instantiate
+(AROB2023) pkg> <delete>
+```
+
+### Python
+Some analyses require Python environment.
```
Comment (ciaran-regan-ie) [inline README-en.md:21]: analysis

**Review by ciaran-regan-ie — CHANGES_REQUESTED** [review]
Great work! A few minor comments, but LGTM!

**Review by ciaran-regan-ie — APPROVED** [review]

**Pull request merged**